mlrun/datastore/sources.py

```diff
@@ -105,14 +108,11 @@ class CSVSource(BaseSourceDriver):
     :parameter path: path to CSV file
     :parameter key_field: the CSV field to be used as the key for events. May be an int (field index) or string
         (field name) if with_header is True. Defaults to None (no key). Can be a list of keys.
-    :parameter time_field: the CSV field to be parsed as the timestamp for events. May be an int (field index) or
-        string (field name) if with_header is True. Defaults to None (no timestamp field). The field will be parsed
-        from isoformat (ISO-8601 as defined in datetime.fromisoformat()). In case the format is not isoformat,
-        timestamp_format (as defined in datetime.strptime()) should be passed in attributes.
+    :parameter time_field: DEPRECATED. Use parse_dates to parse timestamps.
```

Comment:
The code should also provide a deprecation warning if this parameter is used - look for examples of using `PendingDeprecationWarning`